src/native/corehost/fxr/standalone/CMakeLists.txt

```diff
@@ -48,6 +48,16 @@ endif(WIN32)
 
 install_with_stripped_symbols(hostfxr TARGETS corehost)
 
+if (MSVC)
```

Comment:
What is the perf impact of disabling LTCG on our own uses of this library? Do we need two separate builds to avoid de-optimizing out own bits?

Reply:
This static library is not used by this repo so there is no perf impact. The hostcommon static library is used by this repo, however. Is there an easy way to see whether LTCG did anything noteworthy without inspecting the assembly code?

Reply:
These is no easy way to tell.

It does not sound like a good idea to give up LTCG for this.

Reply:
I use MSVC so it doesn't really matter to me. I put the LTCG back.

Reply:
LTCG tends to require exact or very close compiler version match. You cannot just use any MSVC version and assume that it will work.

Disabling LTCG for the shipping build of the .lib sounds like a good idea, but not for the internally used build of the .lib.

Reply:
So what should I do?

1. Create separate public version of libhostcommon.lib and disable LTCG?
2. Create separate "project" for libhostfxr.lib that directly includes source files from hostcommon instead of directly referencing the static library?
3. Something else?

Reply:
2. makes the most sense to me. @vitek-karas @elinor-fung Do you have an opinion?

Reply:
I'll leave it up to @elinor-fung for the final call... I think 2. makes sense for now. Although I would like us to investigate shipping sources instead of libs. That said, the usage of statically linked `hostfxr` outside of this repo is pretty rare as far as I can tell. So maybe it's OK to not have something a bit more complex to consume.

Reply:
I'd agree that (2) makes sense for now.

For shipping sources (or something like it), I think I'd rather move towards vcpkg than directly putting the right sources in a custom package. There's currently a [port for nethost](https://github.com/microsoft/vcpkg/tree/master/ports/nethost) that we don't maintain, but others have been updating (it is at 6.0). I could see adding one for the static `hostfxr` so that consumers could essentially build (2) themselves.